runtime/src/lib.rs

```diff
@@ -203,11 +206,38 @@ parameter_types! {
 	pub const WeightFeeCoefficient: Balance = 1_000;
 }
 
+type NegativeImbalance = <<Runtime as crml_transaction_payment::Trait>::Currency as Currency<
+	<Runtime as frame_system::Trait>::AccountId,
+>>::NegativeImbalance;
+
+pub struct Validators;
+impl OnUnbalanced<NegativeImbalance> for Validators {
+	fn on_nonzero_unbalanced(amount: NegativeImbalance) {
+		let validators = pallet_staking::Module::<Runtime>::current_elected();
+		if validators.len() == 0 {
+			return;
+		}
+		let per_validator_reward = amount.peek() / (validators.len() as Balance);
```

Comment:
[thoughts for later] Will need to think what we want to do with the remainder so that total issuance is correct.

eg, reward = 1000, # validators = 9
reward per validate = 111, remainder = 1

Reply:
My understanding is that the inflation rate that we target at the end of an era would take care of the little discrepancies here.

Reply:
Agree with Alex, just keep it simple and the ignore the remainder will not have effect on the current mechanism